corpus/corpus/settings.py

```diff
@@ -180,3 +182,18 @@
 EMAIL_USE_SSL = True
 
 USE_TAILWIND_CDN_LINK = os.getenv("LIVECYCLE") is not None
+
+SENTRY_DSN = os.environ.get("SENTRY_DSN")
+if os.getenv("ENVIRONMENT", "PRODUCTION") == "PRODUCTION":
+    if SENTRY_DSN:
```

Comment:
In the `else` block, lets raise an error so that the prod environment doesn't start without Sentry

Reply:
Yes, raised an exception